zenoh/src/lib.rs

```diff
@@ -264,6 +264,7 @@ where
 {
     /// Restrict the matching subscribers that will receive data published from this [`Session`]
     /// to the ones that have the given [`Locality`](crate::prelude::Locality).
+    #[cfg(feature = "unstable")]
```

Comment:
`Locality` should not be configurable at session level but only per `declare_subscriber` and `declare_queryable`-basis.